src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -74,17 +79,16 @@ export class ContentTypeFieldsDropZoneComponent extends BaseComponent implements
      */
     saveFieldsHandler(fieldToSave: Field): void {
         const fields = this.getFields();
+        console.log('saveFieldsHandler', fields);
```

Comment:
we already have a logger to use instead of console.logs, there are a few more below